File: docs/reference/snippets/python/tts/voice_generate.py
import requests
import json

url = "https://openapi.surreal-ai.com/v1/tts/voice.generate"

payload = json.dumps({
  "text": "Hello, Surreal Engine",
  "voice_name": "en-US-Wavenet-A"
})
headers = {
  'Authorization': 'Bearer bee03bed6352400383d9a51ba325af2f',
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)

File: docs/reference/snippets/golang/tts/voice_generate.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
	"strings"
)

func main() {
	url := "https://openapi.surreal-ai.com/v1/tts/voice.generate"
	method := "POST"

	payload := strings.NewReader(`{
    "text": "Hello, Surreal Engine",
    "voice_name": "Fabineu Topshot"
}`)

	client := &http.Client{}
	req, err := http.NewRequest(method, url, payload)

	if err != nil {
		fmt.Println(err)
		return
	}
	req.Header.Add("Authorization", "Bearer bee03bed6352400383d9a51ba325af2f")
	req.Header.Add("Content-Type", "application/json")

	res, err := client.Do(req)
	if err != nil {
		fmt.Println(err)
		return
	}
	defer res.Body.Close()

	body, err := ioutil.ReadAll(res.Body)
	if err != nil {
		fmt.Println(err)
		return
	}
	fmt.Println(string(body))
}
